### PR TITLE
Added Privacy Policy and updated footer links

### PR DESCRIPTION
**Pull request by Brend-Smits**
## Description

Added privacy policy and updated the footer links. External links from the footer now open in a new tab instead of on the same page.
Also disabled some footer links, like the ones under 'my profile', as these features have not yet been implemented.

## Type of change

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] This change requires a documentation update

## Checklist

-   [x] My code follows the style guidelines of this project
-   [x] I have performed a self-review of my own code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [x] I have made corresponding changes to the documentation
-   [x] I have added tests that prove my fix is effective or that my feature works
-   [x] I updated the changelog with an end-user readable description
-   [x] I assigned this pull request to the correct project board to update the sprint board

## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Go to home page
2. Click on any of the footer links
3. Click on privacy policy link

## Link to issue

Closes: #211

### CODE REVIEW

File: src/app/components/footer/footer.component.html [hunk 1]
```diff
@@ -1,46 +1,43 @@
...
+          <a href="https://github.com/DigitalExcellence/dex-frontend/wiki" target="_blank"
+            rel="noopener noreferrer">Documentation</a>
+          <a href="mailto:dex.fhict@gmail.com">Contact us</a>
+        </div>
+      </div>
+      <div class="col-md-2">
+        <h4>More info</h4>
+        <div class="menu-items">
+          <a href="https://fontys.nl/" target="_blank" rel="noopener noreferrer">Fontys.nl</a>
+          <a href="https://github.com/DigitalExcellence" target="_blank" rel="noopener noreferrer">Github.com</a>
```
Comment (RensvdLinden) [inline src/app/components/footer/footer.component.html:26]:
```suggestion
          <a href="https://github.com/DigitalExcellence" target="_blank" rel="noopener noreferrer">Github Organization</a>

```

File: src/app/components/footer/footer.component.html [hunk 2]
```diff
@@ -1,46 +1,43 @@
...
+        <h4>Legal</h4>
+        <div class="menu-items">
+          <a routerLink="privacy">Privacy policy</a>
+          <a href="https://github.com/DigitalExcellence/dex-frontend/blob/develop/LICENSE.md" target="_blank" rel="noopener noreferrer">License</a>
+        </div>
+      </div>
+    </div>
+    <div class="row justify-content-center">
+      <div class="col-3 copyright">
+        <p>Find the entire <strong>DeX</strong> project on <a href="https://github.com/DigitalExcellence" target="_blank" rel="noopener noreferrer"><strong>GitHub</strong></a></p>
```
Comment (RensvdLinden) [inline src/app/components/footer/footer.component.html:39]:
```suggestion
        <p>Find <strong>DeX</strong> on <a href="https://github.com/DigitalExcellence" target="_blank" rel="noopener noreferrer"><strong>GitHub</strong></a></p>

```

File: src/app/components/privacy-policy/privacy-policy.component.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
```
Comment (rubenb994) [inline src/app/components/privacy-policy/privacy-policy.component.ts:3]: License is missing & not documentation for component

File: src/app/components/privacy-policy/privacy-policy.component.html [hunk 4]
```diff
@@ -0,0 +1,88 @@
+<h1>Privacy Policy for Digital Excellence</h1>
```
Comment (rubenb994) [inline src/app/components/privacy-policy/privacy-policy.component.html:1]: License is missing

File: src/app/components/footer/footer.component.html [hunk 5]
```diff
@@ -1,46 +1,45 @@
...
-      </div>
-    </div>
-    <div class="row justify-content-center">
-      <div class="col-3 copyright">
-        <p>Find the entire <b>DeX</b> project on <a href="https://github.com/DigitalExcellence"><b>GitHub</b></a></p>
-      </div>
-    </div>
-  </div>
-</footer>
+<footer>
```
Comment (rubenb994) [inline src/app/components/footer/footer.component.html:1]: License is missing

Comment (Brend-Smits) [reply src/app/components/footer/footer.component.html:20]: Added, thanks!

File: src/app/components/privacy-policy/privacy-policy.component.html (hunk 4 above)
Comment (Brend-Smits) [reply src/app/components/privacy-policy/privacy-policy.component.html:19]: Added, thanks!

File: src/app/components/privacy-policy/privacy-policy.component.ts (hunk 3 above)
Comment (Brend-Smits) [reply src/app/components/privacy-policy/privacy-policy.component.ts:24]: Added, thanks!

**Pull request merged**